client/components/GMap.js

```diff
@@ -137,12 +174,7 @@ class GMap extends Component {
   }
 
   createOrUpdateUserCircle (map, position) {
-    if (this.state.userCircle) {
-      this.state.userCircle.inner.setCenter(position)
-      this.state.userCircle.outer.setCenter(position)
-    } else {
-      this.createUserCircle(map, position, 402)
-    }
+    this.createUserCircle(map, position, 402)
```

Comment:
We need to be able to update the position of the UserCircle. In order to do that, we need to preserve the check for the userCircle, then setPosition of the circle markers. Please put back the original code w/ the necessary modifications.